### PR TITLE
feat(dAppStore): Move external apps into external sites

### PR DESCRIPTION
**Pull request by omniwired**
image

### CODE REVIEW

File: packages/screens/DAppStore/components/DAppStoreData.tsx [hunk 1]
```diff
@@ -1,91 +0,0 @@
-import { useQuery } from "@tanstack/react-query";
```
Comment (omniwired) [inline packages/screens/DAppStore/components/DAppStoreData.tsx:1]: @n0izn0iz I'm sure you will love this one.

**Comment by omniwired** [comment]
since the idea behind this change is to allow developers to add their apps in the dApp Store.
It makes no sense to keep the airtable implementation around.

File: packages/screens/DAppStore/query/getFromFile.ts [hunk 2]
```diff
@@ -164,14 +162,14 @@ export function getAvailableApps(): dAppGroup {
...
         },
       },
     },
-    "coming-soon": {
-      id: "coming-soon",
-      groupName: "Coming soon dApps",
+    bookmarks: {
+      id: "bookmarks",
+      groupName: "External Apps",
       icon: otherAppsIcon,
```
Comment (WaDadidou) [inline packages/screens/DAppStore/query/getFromFile.ts:173]: The icon for "coming-soon" and  "bookmarks" doesn't appear 
image

**Comment by WaDadidou** [comment]
Some icons look weird in the Sidebar
image

File: packages/components/navigation/Sidebar.tsx [hunk 3]
```diff
@@ -124,7 +124,13 @@ export const Sidebar: React.FC = () => {
           return (
             <SidebarButton
               key={item.id}
-              onPress={onRouteChange}
+              onPress={
+                route === "External"
```
Comment (WaDadidou) [inline packages/components/navigation/Sidebar.tsx:128]: Maybe you can type `route` from `getFromFile.ts` with something like `"External" | "ComingSoon" | keyof RootStackParamList`

File: packages/components/ScreenContainer/index.tsx [hunk 4]
```diff
@@ -67,6 +69,14 @@ export const ScreenContainer: React.FC<{
   forceNetworkKind,
   forceNetworkFeatures,
 }) => {
+  const dispatch = useAppDispatch();
```
Comment (n0izn0iz) [inline packages/components/ScreenContainer/index.tsx:72]: can we move this to a component in App.tsx?
(see WalletSyncer component)

File: packages/components/navigation/Sidebar.tsx (hunk 3 above)
Comment (omniwired) [reply packages/components/navigation/Sidebar.tsx:128]: thanks for the code; 
Updated!

File: packages/components/ScreenContainer/index.tsx (hunk 4 above)
Comment (omniwired) [reply packages/components/ScreenContainer/index.tsx]: done; thanks for the feedback

**Pull request merged**